rollup.config.js

```diff
@@ -1,10 +1,13 @@
-import typescript from "@rollup/plugin-typescript";
+import ts from "@wessberg/rollup-plugin-ts";
```

Comment:
This plugin is zero config and takes care of TS compilation and spits out type definitions nicely